### PR TITLE
fix: unrevert verbiage changes

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Restore the verbiage changes that were reverted in #45347

### CODE REVIEW

**Review by ojeytonwilliams — DISMISSED** [review]
LGTM :+1:

File: curriculum/challenges/english/13-relational-databases/build-a-number-guessing-game-project/build-a-number-guessing-game.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ This is one of the required projects to earn your certification. For this projec
 
 # --instructions--
 
-**Important:** After you pass all the project tests, save a dump of your database into a `number_guessers.sql` file, as well as your whole `number_guessing_game` folder, so you can complete step 2. There will be instructions how to do that within the virtual machine.
+**Important:** After you pass all the project tests, save a dump of your database into a `number_guess.sql` file, as well as your whole `number_guess.sh` file, so you can complete step 2. There will be instructions how to do that within the virtual machine.
```
Comment (moT01) [inline curriculum/challenges/english/13-relational-databases/build-a-number-guessing-game-project/build-a-number-guessing-game.md:16]:
```suggestion
**Important:** After you pass all the project tests, save a dump of your database into a `number_guess.sql` file, as well as your `number_guess.sh` file, so you can complete step 2. There will be instructions how to do that within the virtual machine.
```

**Review by moT01 — COMMENTED** [review]
One sort of typo I missed on the initial PR

**Review by moT01 — COMMENTED** [review]
Looks like cypress had an issue with a few tests as well.
[This one for sure](https://github.com/freeCodeCamp/freeCodeCamp/pull/45347/files#diff-1fe1691f7888d87fb2d388f444d38cd0f510b0f4d1b7cbcf34b7c6c193da9126R14), and [maybe this one](https://github.com/freeCodeCamp/freeCodeCamp/pull/45347/files#diff-d177c638e563c1c8929dfc5fb477a70c67613fd5736eaec01fb0f3d24c27f99bR20)

**Review by moT01 — DISMISSED** [review]
LGTM 🎉

File: curriculum/challenges/english/13-relational-databases/build-a-number-guessing-game-project/build-a-number-guessing-game.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ This is one of the required projects to earn your certification. For this projec
 
 # --instructions--
 
-**Important:** After you pass all the project tests, save a dump of your database into a `number_guessers.sql` file, as well as your whole `number_guessing_game` folder, so you can complete step 2. There will be instructions how to do that within the virtual machine.
+**Important:** After you pass all the project tests, save a dump of your database into a `number_guess.sql` file, as well as your `number_guess.sh` file, so you can complete step 2. There will be instructions how to do that within the virtual machine.
```
Comment (Sembauke) [inline curriculum/challenges/english/13-relational-databases/build-a-number-guessing-game-project/build-a-number-guessing-game.md:16]:
```suggestion
**Important:** After you pass all the project tests, save a dump of your database into a `number_guess.sql` file, as well as your `number_guess.sh` file, so you can complete step 2. There will be instructions on how to do that within the virtual machine.
```

File: curriculum/challenges/english/13-relational-databases/build-a-periodic-table-database-project/build-a-periodic-table-database.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ This is one of the required projects to earn your certification. For this projec
 
 # --instructions--
 
-**Important:** After you pass all the project tests, save a dump of your database into a `elements.sql` file, as well as your whole `periodic_table` folder, so you can complete step 2. There will be instructions how to do that within the virtual machine.
+**Important:** After you pass all the project tests, save a dump of your database into a `periodic_table.sql` file, as well as your `element.sh` file, so you can complete step 2. There will be instructions how to do that within the virtual machine.
```
Comment (Sembauke) [inline curriculum/challenges/english/13-relational-databases/build-a-periodic-table-database-project/build-a-periodic-table-database.md:16]:
```suggestion
**Important:** After you pass all the project tests, save a dump of your database into a `periodic_table.sql` file, as well as your `element.sh` file, so you can complete step 2. There will be instructions on how to do that within the virtual machine.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**